test/qr.jl

```diff
@@ -27,20 +27,19 @@ srand(42)
         @test Q'*Q ≈ one(Q'*Q)
         @test istriu(R)
 
-        # # pivot=true cases are not released yet
-        # pivot = Val(true)
-        # QRp = @inferred qr(arr, pivot)
-        # @test QRp isa Tuple
-        # @test length(QRp) == 3
-        # Q, R, p = QRp
-        # @test Q isa StaticMatrix
-        # @test R isa StaticMatrix
-        # @test p isa StaticVector
+        # pivot=true cases are not released yet
```

Comment:
Is this comment true/helpful at this point?

Reply:
I don't know what it means -- perhaps that there is not released StaticArray-specific version and that we just call to LAPACK? I can add that instead

Reply:
I don't know either. Looks like it was added by @denius in https://github.com/JuliaArrays/StaticArrays.jl/pull/328. Probably your interpretation is right.